lib/utilityservice.js

```diff
@@ -1304,3 +1304,13 @@ UtilityService.prototype.ignoreTerminalPrompt = async function (dirname) {
     childProcess.execSync(`cd ${dirname}`);
     process.env.GIT_TERMINAL_PROMPT = 0;
 }
+
+UtilityService.prototype.createVisualForceOrgUrl = function (urlTemplate, datapackId) {
+    let designerPath = '';
```

Comment:
Should probably move all the logic for creating the Url here. It should just send the dataPackData here and then return the fully formed Url from this single method. Since otherwise there is duplicate code to get the urlTemplate and dataPackId.